Cue/app/tabs/library/LibraryHome.js

```diff
@@ -111,7 +111,16 @@ class LibraryHome extends React.Component {
       [
         {text: 'Cancel', style: 'cancel'},
         {text: 'Create', onPress: (deckName) => {
-          if (deckName && deckName.length) {
+          if (!deckName || !deckName.length) {
+            Alert.alert(
+              Platform.OS === 'android' ? 'Enter a deck name' : 'Enter a Deck Name',
```

Comment:
-----
**Deck Name Can’t Be Empty**

A good deck name is short but descriptive.

-----
? I'm not sure a second sentence is really necessary though. Need a second opinion.